app/src/main/java/fr/free/nrw/commons/wikidata/WikidataEditService.kt

```diff
@@ -37,7 +37,6 @@ import javax.inject.Inject
 import javax.inject.Named
 import javax.inject.Singleton
 
-
```

Comment:
Please perform such stylistic changes in a separate style-only pull request. Thanks!